models/UserManager.php

```diff
@@ -0,0 +1,289 @@
+<?php
+
+class UserManager extends Model
+{
+    // Fonction permettant de récupérer le nom d'un admin pour un article précis.
+    function article_admin(){
+     $req = $this->db->query("
+            SELECT  
+                    admins.name
+            FROM articles
+            JOIN admins
+            ON articles.writer=admins.email
+            WHERE articles.id = '{$_GET['id']}'
+
+        ");
+        
+        $admin = [];
+        
+        while($row = $req->fetch()){
+            $admin   = new User();
+            $admin   ->setName($row['name']);
+               
+            $admins[] = $admin;
+        }
+        return $admin;
+    }
+    
+    // Fonction permettant de vérifier que la session est bien active sous le role administrateur
+    function admin(){
+        if(isset($_SESSION['admin'])){
+            $tableau = [
+                'email'     =>  $_SESSION['admin'],
+                'role'      =>  'admin'
+            ];
+
+            $sql = "SELECT * FROM admins WHERE email=:email AND role=:role";
+            $req = $this->db->prepare($sql);
+            $req->execute($tableau);
+            $exist = $req->rowCount($sql);
+
+            return $exist;
+        }else{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method admin uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)